### PR TITLE
spelling modification

### PR DESCRIPTION
**Pull request by dengyufeng2206**
Signed-off-by: dengyufeng2206 <deng.yufeng@zte.com.cn>

#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
modify incorrect words.
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by dengyufeng2206** [comment]
I signed it!

File: staging/src/k8s.io/client-go/util/certificate/certificate_manager.go [hunk 1]
```diff
@@ -126,7 +126,7 @@ type Config struct {
 	// allows one to setup monitoring and alerting of unexpected rotation
 	// behavior and track trends in rotation frequency.
 	CertificateRotation Histogram
-	// CertifcateRenewFailure will record a metric that keeps track of
+	// CertificateRenewFailure will record a metric that keeps track of
```
Comment (heqg) [inline staging/src/k8s.io/client-go/util/certificate/certificate_manager.go:129]: looks good to me

**Comment by aramase** [comment]
Thank you for the PR! It seems to only contain trivial edits. Please read our [trivial edits policy](https://github.com/kubernetes/community/blob/master/contributors/guide/pull-requests.md#trivial-edits). We encourage you to take a look at confirmed issues and bugs or issues marked help-wanted.

/close

**Pull request closed without merging**